src/current/v23.1/migration-overview.md

```diff
@@ -5,89 +5,351 @@ toc: true
+
+- {% include {{ page.version.version }}/migration/load-data-import-into.md %} Typically, initial data loading during a database migration will not be running concurrently with application traffic, so the fact that `IMPORT INTO` takes the table offline may not have any observable availability impact.
+- {% include {{ page.version.version }}/migration/load-data-third-party.md %} Within the tool, you can select the database tables to migrate to the test cluster.
+- {% include {{ page.version.version }}/migration/load-data-copy-from.md %}
+
+#### Validate queries
+
+After you [load the test data](#load-test-data), validate your queries on CockroachDB. You can do this by [shadowing](#shadowing) or by [manually testing](#test-query-results-and-performance) the queries.
+
+##### Shadowing
+
+You can "shadow" your production workload by replicating all writes from the source database to CockroachDB. To do this, use a [change data capture (CDC)](cdc-queries.html) service. {% comment %}The CockroachDB Live Migration Service (MOLT LMS) can also write queries to both databases simultaneously.{% endcomment %} You can then [test the queries](#test-query-results-and-performance) on CockroachDB for consistency, performance, and potential issues with the migration.
+
+Shadowing may not be necessary or practical for your workload. For example, because transactions are serialized on CockroachDB, this will limit your ability to validate the performance of high-throughput workloads.
+
+##### Test query results and performance
+
+You can manually validate your queries by testing a subset of "critical queries" on an otherwise idle CockroachDB cluster:
+
+- Check the application logs for error messages and the API response time. If application requests are slower than expected, use the **SQL Activity** page on the [{{ site.data.products.db }} Console](../cockroachcloud/statements-page.html) or [DB Console](ui-statements-page.html) to find the longest-running queries that are part of that application request. If necessary, tune the queries according to our best practices for [SQL performance](performance-best-practices-overview.html).
+
+- Compare the results of the queries and check that they are identical in both the source database and CockroachDB. To do this, you can use [MOLT Verify](molt-verify.html).
+
+Test performance on a CockroachDB cluster that is appropriately [sized](#capacity-planning) for your workload:
+
+1. Run the application with single- or very low-concurrency and verify the app's performance is acceptable. The cluster should be provisioned with more than enough resources to handle this workload, because you need to verify that the queries will be fast enough when there are zero resource bottlenecks.
```

Comment:
The phrase `single-` seems incomplete

Reply:
It's abbreviating "single-concurrency or very low-concurrency".